austation/code/modules/mining/equipment/marker_beacons.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/stack/marker_beacon/attack_self(mob/user)
+	if(GLOB.total_beacons >= GLOB.max_beacons) //check that  there are not 100 in the world already
+		to_chat(user, "<span class='warning'>You can not place another beacon!</span>")
```

Comment:
can you give minebots the ability to pick up beacons too?

Reply:
Honestly just remove the limit. It doesn't make sense.
Just give the minebots a finite internal supply of beacons, and let them pick the beacons up again to re-use.

Reply:
Beacons are very easy to come by, as either mob.
This limit was put in place to prevent players from placing hundreds of lighting objects on the world.
I can remove it if you're sure.

Reply:
I'm sure. this has never happened before with human miners, because you have to buy them with mining credits. It becomes an issue when drones have an infinite supply at a click of a single HUD button.

Reply:
removed limit